### PR TITLE
Feature/add food items

### PR DESCRIPTION
**Pull request by jackowiak**

### CODE REVIEW

File: webapp/src/routes/home/game/elements/animal.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { Container } from 'pixi.js';
 
-import { FENCES_INITIAL_Y, FENCES_ROWS, ANIMALS } from '../game.constans';
+import { FENCES_INITIAL_Y, FENCES_ROWS, ANIMALS, FOOD } from '../game.constans';
```
Comment (mkmieciak) [inline webapp/src/routes/home/game/elements/animal.js]: game.constants

File: webapp/src/routes/home/game/elements/foodFence.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    this.stage.x = position.x;
+    this.stage.y = position.y;
+    if (this.isEven(column)) {
+      this.flipHorizontally(this.stage);
+    }
+  }
+
+  isEven = num => num % 2;
+
+  flipHorizontally(sprite) {
```
Comment (mkmieciak) [inline webapp/src/routes/home/game/elements/foodFence.js]: static flipHorizontally

File: webapp/src/routes/home/game/elements/foodFence.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    this.stage.width = 39;
+
+    this.stage.x = position.x;
+    this.stage.y = position.y;
+    if (this.isEven(column)) {
+      this.flipHorizontally(this.stage);
+    }
+  }
+
+  isEven = num => num % 2;
```
Comment (mkmieciak) [inline webapp/src/routes/home/game/elements/foodFence.js]: isEven = num => num % 2 === 0;

File: webapp/src/routes/home/game/elements/foodFenceGroup.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+export class FoodFenceGroup {
+  constructor({ rendererWidth, rendererHeight }) {
+    this._stage = new Container();
+
+    this.renderFences(rendererWidth, rendererHeight);
+  }
+
+  renderFences = (rendererWidth, rendererHeight) => {
+    const firstColumnXPos = 130;
```
Comment (mkmieciak) [inline webapp/src/routes/home/game/elements/foodFenceGroup.js]: you can assign these values in constructor

**Comment by m1chalkubiak** [comment]
Taka ogólna uwaga/sugestia o ewentualny refactor 
https://www.codeandweb.com/texturepacker/tutorials/how-to-create-sprite-sheets-and-animations-with-pixijs

**Pull request merged**